src/snyk/common/languageServer/settings.ts

```diff
@@ -20,6 +20,7 @@ export type ServerSettings = {
   manageBinariesAutomatically?: string;
   cliPath?: string;
   token?: string;
+  filterSeverity?: SeverityFilter;
```

Comment:
the ever present question: is this a pure initialization option or a config? And should it be part of the ServerSettings object or not?

Reply:
`SeverityFilter` is a configuration option that will be passed into LS on startup, is there a preferred way to do it other than via in `ServerSettings`. The other way I see is to extend the `InitializationOptions` and pass it directly, but I thought using the `LanguageServerSettings.fromConfiguration(this.configuration);` for serialization was nicer.

Reply:
>  is this a pure initialization or a config?
It's a config, since this the setting can be changed during runtime 🪂